docs/HOWTO_DEPLOYING_AWS_LAMBDA.md

```diff
@@ -31,7 +31,7 @@ Waiting for verification for a certificate may take a while so we will start tha
 
 ## S3
 
-Create a private S3 bucket by choosing all the default values as you go throuh the setup wizard. We will call this `textforcampaign`.
+Create a private S3 bucket by choosing all the default values as you go through the setup wizard. We will call this `textforcampaign`. Make sure the bucket is publicly accessible.
```

Comment:
Is it worth noting here that thre s3 bucket used for S3_STATIC_PATH and STATIC_BASE_URL (same one) should definitely be publicly accessible.

As a policy, AWS_S3_BUCKET_NAME does need to allow objects to be publicly accessible (basically turn OFF 'block public access'), however there shouldn't be a blanket visibility -- in fact that would be very bad for security.
MoveOn doesn't use the same s3 bucket for all of those things (we have two -- one for the public code and one for the backend files (exports, and s3 contact loader)) but I suppose that's not necessary since the package.json `prod-static-upload` command adds acl-public